src/shell/meanbee/configpoweredcss/regenerate.php

```diff
@@ -0,0 +1,47 @@
+<?php
+require_once __DIR__ . '/../../abstract.php';
+
+class Meanbee_Configpoweredcss_Regenerate extends Mage_Shell_Abstract
+{
+
+    /**
+     * Run script
+     *
+     */
+    public function run()
+    {
+
+        /**
+         * Shell script regenerate the configpowered css for all stores
+         */
+
+        try {
+
+            /** @var Meanbee_ConfigPoweredCss_Model_Css $css */
+            $css = Mage::getModel('meanbee_configpoweredcss/css');
+
+            $stores = Mage::app()->getStores();
+            foreach ($stores as $storeId => $store) {
```

Comment:
I don't require it to be changed for this to be merged but this is heavily repeating code that's in the controller. That would tell me that it's worth pulling this for loop out into a function on a helper. The controller and this shell command would then just catch exceptions and handle them in their own way. Something to consider.